app/models/git_hub_api.rb

```diff
@@ -14,7 +14,10 @@ def self.execute(client, cmd, *args)
     val = nil
     t = Benchmark.realtime { val = client.send(cmd, *args) }
     limit_after = client.rate_limit.remaining
-    logger.info("Executed #{cmd} #{args.inspect}...api calls remaining #{limit_after} (in #{"%0.3f" % t}s using #{limit_before - limit_after} calls)")
+    logger.info("Executed #{cmd} #{args.inspect}...api calls remaining #{limit_after} " \
```

Comment:
The fact that it's multiline, or the continued parsing with `\`? (would you prefer `+`?) I'm not a fan of lines with an enormous amount of columns.